src/RequestParser.php

```diff
@@ -133,7 +133,7 @@ public function parseBody($content)
                 return;
             }
 
-            if (strtolower($headers['Content-Type']) == 'application/x-www-form-urlencoded') {
+            elseif (strpos(strtolower($headers['Content-Type']), 'application/x-www-form-urlencoded') === 0) {
```

Comment:
Fair enough, leave it as is. Was mostly nitpicking because I've seen clients (and servers) pull really odd stunts so trying to be a bit more defensive and prepared for such occasions.